### ISSUE TITLE
Shut down or restart function

### ISSUE THREAD

**Issue by retiep** [crieke/RoonServer_QNAP_Installer#8]
Would it be possible to add a restart function in the menu of the server? Now the only way to do this is through the download latest software button.

**Comment by crieke** [comment]
Yes, a restart of Roon Server is required when the the latest version should be installed from the UI (or when selecting a different the database location).  Implementing a restart button seems like a redundant thing to me and was only used in the UI when it is absolutely required.
For a simple restart, you can use stop/start button in the App Center.
